duckduckgo/macos-browser
# Use manual code signing in Review and Release build configurations

Task/Issue URL: https://app.asana.com/0/1201037661562251/1203479115456657/f

**Description**:
Switching to manual code signing allows us to delete CI_Review and CI_Release build configurations.
CI builds are not affected, and to build locally all you need to do is:
* Go to Xcode settings -> Accounts
* Select DDG Apple ID and _DuckDuckGo, Inc._ team
* Click _Download Manual Profiles_
After that, you're all set until the Developer ID Application certificate expires (which is in 18 years).



**Steps to test this PR**:
1. Make sure that Review and Release builds work when triggered from Xcode via Main Menu -> Product -> Archive
1. Make sure that `./script/archive.sh review` and `./script/archive.sh release` work (note: you need to set up app-specific password for your Apple ID to get `archive.sh` to work locally).

I have verified that GHA works fine with both [review](https://github.com/more-duckduckgo-org/macos-browser/actions/runs/3600094626) and [release](https://github.com/more-duckduckgo-org/macos-browser/actions/runs/3600095622) builds.

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
